country-currency-map.js

```diff
@@ -26,7 +27,20 @@ function formatCurrency(value, currencyAbbr) {
     return `${value} ${currencyAbbr}`;
 }
 
+function getCurrencyList() {
+    return currencyMap;
+}

```

Comment:
Should this instead return the list of currency names (and sorted) since that's how it would be displayed in a dropdown?